mrmap/registry/tests/acceptance/features/api/metadata.datasets.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Dataset Metadata
+  As an API client
+  I want to control the filtering of resources
+  so that only resources are part of the response that are founded by the given filter.
+
+  Scenario: Visiting services API without any search criteria, will return every feature
+    Given an authorized session
+    Given a request url http://localhost:8000/api/v1/registry/metadata/datasets/?ordering=title
+    When the request sends GET
+    Then the response status is OK
+    And all dataset_metadata objects are returned
+    And a paginated response is returned
+    And response objects are ordered ascending by title
+    And "accessible" key is part of the response
+
+
+  Scenario: Only resources which has keyword, title, abstract or uuid like "WMS" is part of the response
```

Comment:
Since they are all words that are part of `search` and not separate filters, it does not matter. Right now, doing `Given a request url http://localhost:8000/api/v1/registry/metadata/datasets/?search=WMS&ordering=title` is actually searching for records that have keywords, title, abstract or ID that contain the word `WMS`

I would agree to that if we add specific data to be found in each case, loaded by the fixtures

Reply:
Maybe it would be better to have different services, but can't we simply match the same WMS instance in separate scenarios using a different type of match? IMHO, this should still be an improvement.